www/js/app.js

```diff
@@ -29,34 +30,58 @@ var app = angular.module('starter',
                 templateUrl: 'templates/create_booking.html',
                 controller: 'CreateBooking'
             })
-            .state('resourceDetails', {
-                url: '/:type/:id',
-                templateUrl: 'templates/details.html',
-                controller: 'ResourceDetails'
-            })
+            // .state('resourceDetails', {
+            //     url: '/:type/:id',
+            //     templateUrl: 'templates/details.html',
+            //     controller: 'ResourceDetails'
+            // })
             .state('searchCar', {
                 url: '/:type/:id/cars',
                 templateUrl: 'templates/search_car.html',
                 controller: 'SearchCar'
             })
-            .state('home', {
-                url: '/',
-                templateUrl: 'templates/home.html',
-                controller: 'Home'
-            })
             .state('resourceList', {
                 url: '/manage',
                 templateUrl: 'templates/resources.html',
                 controller: 'ResourceList'
             })
-            .state('settings', {
+
+            /** Tab bar routes **/
+            .state('tabsController.home', {
+                url: '/homePage',
```

Comment:
Convention is to have the landing page either on route `/` or `/home` or `/index`. I personally prefer `/`, however feel free to update to fit any of these conventions.